lib/rspec/core/memoized_helpers.rb

```diff
@@ -1,3 +1,5 @@
+require 'thread'
```

Comment:
Yeah, I like that. Someone somewhere will have a `Queue` class will have half its methods overridden by the one in thread, and then their program will segfault, just b/c we require thread here >.< I'll do something similar to this. Means 1.8 won't be threadsafe, but 1.8's been end of lifed, so shouldn't be any new development on it, anyway.